activities-examples/src/workflows.test.ts

```diff
@@ -12,19 +12,15 @@ beforeAll(async () => {
     logger: new DefaultLogger('WARN', (entry: LogEntry) => console.log(`[${entry.level}]`, entry.message)),
   });
 
-  testEnv = await TestWorkflowEnvironment.create({
-    testServer: {
-      stdio: 'inherit',
-    },
```

Comment:
@bergundy I forget—is this now default behavior, or is there another equiv option?

Reply:
There’s no output from the test server so this doesn’t matter in practice, for temporalite we allow configuring the log level